projects/ngrx.io/content/examples/router-store-selectors/src/app/car/car.component.ts

```diff
@@ -2,11 +2,14 @@
 import { Component } from '@angular/core';
 import { Store } from '@ngrx/store';
 import { selectCar } from './car.selectors';
+import { AsyncPipe, JsonPipe } from '@angular/common';
 
 @Component({
+  standalone: true,
   selector: 'app-car',
   templateUrl: './car.component.html',
   styleUrls: ['./car.component.css'],
+  imports: [AsyncPipe, JsonPipe],
 })
 export class CarComponent {
   car$ = this.store.select(selectCar);
```

Comment:
Can you also refactor this to use `inject` instead of the constructor please?

Reply:
Done